### PR TITLE
feat(cross-chain-oracle): Add Nomad adapters

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

I've tested these contracts successfully in a staging environment and sent Oracle and Governance messages between Rinkeby and Kovan so far. The other supported networks so far are Moonbeam and Celo's testnets.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [X]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/core/contracts/cross-chain-oracle/README.md [hunk 1]
```diff
@@ -1,4 +1,34 @@
-# Deployments
+# Cross Chain Infrastructure
```
Comment (chrismaree) [inline packages/core/contracts/cross-chain-oracle/README.md:1]: nice. great extra docs.

File: packages/core/contracts/cross-chain-oracle/chain-adapters/Nomad_ChildMessenger.sol [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    event MessageReceivedFromParent(
+        address indexed targetSpoke,
+        bytes dataToSendToTarget,
+        uint32 sourceDomain,
+        address sourceSender
+    );
+
+    /**
+     * @notice Only accept messages from a Nomad Replica contract
+     */
```
Comment (chrismaree) [inline packages/core/contracts/cross-chain-oracle/chain-adapters/Nomad_ChildMessenger.sol:35]: might be better to use the `//` comment standard for this comment?

File: packages/core/contracts/cross-chain-oracle/chain-adapters/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## Configuration Details
+
+Setting up contracts for each type of network are different.
+
```
Comment (chrismaree) [inline packages/core/contracts/cross-chain-oracle/chain-adapters/README.md:4]: nice. shall we add the same block for other chains?

Comment (nicholaspai) [reply packages/core/contracts/cross-chain-oracle/chain-adapters/README.md:4]: Yeah we should but i wanted to kick it off here and align on structure of this doc

File: packages/core/contracts/cross-chain-oracle/chain-adapters/Nomad_ChildMessenger.sol (hunk 2 above)
Comment (nicholaspai) [reply packages/core/contracts/cross-chain-oracle/chain-adapters/Nomad_ChildMessenger.sol:35]: done

**Comment by chrismaree** [comment]
Shall we close this PR?

**Pull request closed without merging**